addons/hr_timesheet/wizard/hr_employee_delete_wizard_views.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record id="hr_employee_delete_wizard_form" model="ir.ui.view">
+            <field name="name">Delete Employee</field>
+            <field name="model">hr.employee.delete.wizard</field>
+            <field name="arch" type="xml">
+                <form string="Delete Employee">
+                    <field name="has_timesheet" invisible="1"/>
+                    <field name="has_active_employee" invisible="1"/>
+                    <field name="employee_count" invisible="1"/>
+                    <span attrs="{'invisible': [('has_timesheet', '=', False)]}">
+                        You cannot delete an employee<span attrs="{'invisible': [('employee_count', '&lt;=', 1)]}">s</span>who has timesheets.
```

Comment:
```suggestion
                        You cannot delete employees who have timesheets.
```
We don't support plurals/singular (mainly because it causes translation issues, I think).
If we supported it, "an" would also depend on `employee_count` and it gets messy real quick.